tests/fuzz/fuzz-bip32.c

```diff
@@ -0,0 +1,52 @@
+#include <assert.h>
+#include <stdint.h>
+#include <string.h>
+#include <tests/fuzz/libfuzz.h>
+
+#include <ccan/tal/tal.h>
+#include <common/bip32.h>
+#include <wally_bip32.h>
+
+void init(int *argc, char ***argv)
+{
+}
+
+void run(const uint8_t *data, size_t size)
+{
+	struct ext_key xkey;
+	struct bip32_key_version version;
+	uint8_t *wire_buff;
+	const uint8_t **xkey_chunks, **ver_chunks, *wire_ptr;
+	size_t wire_max;
+
+	if (size < BIP32_SERIALIZED_LEN)
+		return;
+
+	xkey_chunks = get_chunks(NULL, data, size, BIP32_SERIALIZED_LEN);
+	for (size_t i = 0; i < tal_count(xkey_chunks); i++) {
+		wire_max = tal_count(xkey_chunks[i]);
```

Comment:
`tal_bytelen`?

Reply:
Why is it better ? (not bikeshedding, doing it but just to get the rationale)

Reply:
It's more explicit imo -- you're looking for the byte count here